### DISCUSSION TITLE
3.1.4. Cannot load import

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#765] [Q&A]
Haven't gotten very far with this. Installed it on staging site and when I tried to move to the Settings or Spreadsheet Example & Spec received this error on a blank page: Cannot load import. 

As with all other updates to this plugin, it would be helpful if you could give us a heads-up the new version is out and what we can expect. Yes, I did look at the details before I updated, but it was not very descriptive on how to use. Comments below,

Where is this switch UI feature to facilitate switching between the two available user interface displays: Legacy UI and TSML UI?

What is Refactor Import & Settings page with tabs & cards to segregate and group features and settings?

What does this mean to modify feed to follow the directive from setting “Meeting/Group Contacts Are” (Private/Public)?

How is this improved for CSV export/import, includes contact and imported feed information?

**Comment by kiyote33** [comment]
Hi @Grateful-Viv, I'm happy to try and provide some clarity for you.
The **Switch UI** feature simply provides an easy way for Admins to change the User Interface that the public sees from the standard UI (that we call Legacy UI) to the new one called TSML UI. The default is Legacy UI, but we do encourage everyone to give TSML UI a look and see if it's a fit for you and your website.
**Refactor Import & Settings** is pretty much a simple reorganization of that page, adding tabs to segregate Import/Export features from Settings configurations and the Example & Spec detail. The features that existed before are still there, mostly unchanged in their use and behavior. The Switch UI and About Us info cards are new to this version and the Need Help card now has a 2nd linked button which takes you directly to this forum.
**Meeting/Group Contacts Are** is a typo that we didn't catch before publishing. After refactoring, that heading is now "**Contact Visibility**".  I believe what is meant by **follow the directive** is to ensure the code follows the directive set by the Contact Visibility setting (either public or private) and toggles the visibility of the contact information fields on the meeting detail screen accordingly. The default is private so those contact information fields should not display unless this setting is changed to public.
Your last question I'm unsure of exactly what was done to "**improved for CSV export/import**". Maybe one of the maintainer can pipe in with some clarity on that. I do know that we did include some new Q & A in the readme.txt file providing information on the Change Detection feature introduced in version 3.13.
Hope this provides some of the clarity you we're looking for...

**Reply by Grateful-Viv** [reply]
Thanks for the clarifying some of my questions. I am not able to see the Switch UI or other enhancements -- maybe it's because of the error message that comes up when I try to open these other two tabs. Over a year ago, I did extensive testing of TSML UI and was very impressed with it. Many of the suggestions I asked for were even implemented (thanks @joshreisner). But the only reasons I hadn't switched over was because I wanted the sidebar widgets to open the UI (not the Legacy) and the version at least I was looking at (which I've repeatedly looked at at least every month to this date) is it seemed to be always changing. That's great in a test environment, but I was hoping for a locked down version that I could test out before implementing it on the live site.

**Reply by kiyote33** [reply]
@Grateful-Viv Your most welcome. 
It would be great if you could provide details of your environment so we can try to replicate the "Cannot load import" error and provide a resolution. I'm sure you can appreciate the difficulty we face in trying to diagnose an issue which isn't happening in our own sandbox.

**Reply by Grateful-Viv** [reply]
WP 5.9.3
PHP 7.4.28

Disabled all plugins and one by one activated them until I found the culprit. It's a plugin called Ozh' Admin Drop Down Menu which gives a cleaner UI to the WP dashboard by putting the menu bar at the top rather than on the left side.

Is there an absolute need to have these three screens (Import, Settings, Spreadsheet & Spec)? It was fine on one page. Did someone complain it was too cluttered? The settings box could have had the items separated on the old version just as you did in the new one so it wouldn’t take up so much horizontal space. 

Old:
![Import-Settings-‹-Southern-Delaware-Intergroup-—-WordPress](https://user-images.githubusercontent.com/62247421/163673300-fafffca5-998a-4928-aa4f-64ec9aa1b920.png)

New:
![Import-Settings-‹-Staging-Site-—-WordPress](https://user-images.githubusercontent.com/62247421/163673307-43961019-c6da-4347-b271-4d7974e4a1fb.png)
